debian/zim-tools.install

```diff
@@ -1 +1 @@
-usr/bin/
+usr/bin/*
```

Comment:
Ubuntu packaging step is failing due to this path expression. Let's try running locally to use the correct expression here.

Reply:
I have fixed this regression I have added this regression